01-backup.md

```diff
@@ -54,6 +54,23 @@ we're telling Git:
 
 The four commands above only need to be run once:
 the flag `--global` tells Git to use the settings for every project on this machine.
+The configuration options get stored in your home directory in a text file called `.gitconfig`.
+
+~~~ {.bash}
+$ cat ~/.gitconfig
+~~~
+~~~ {.output}
+[user]
+	name = Vlad Dracula
+	email = vlad@tran.sylvan.ia
+[color]
+	ui = true
+[core]
+	editor = nano
+~~~
+
+This file can be edited directly.
```

Comment:
On Sat, Feb 07, 2015 at 01:44:06PM -0800, Raniere Silva wrote:
> > +This file can be edited directly.
> 
> What about
> 
> ~~~
> This file can be edited directly **but you should not do it** to avoid problems.
> ~~~

I frequently edit mine by hand ;).  I'm fine with a warning, but
bolding the warning seems a bit alarmist.  I think there's enough FUD
about Git out there, so there's no need to add more.  It's hard to
imagine anyone breaking their .gitconfig so much that it actually
causes damage (although I can imagine folks breaking it enough to
cause confusion ;).